packages/icons/icons.json

```
@@ -1745,217 +1745,217 @@
     },
     {
         "displayName": "Text highlight",
         "iconName": "text-highlight",
-        "tags": "selector, content",
+        "tags": "selector, content, type, input, text insert, select, highlight, text input, text box, input text, search, enter text, text area",
         "group": "editor",
         "codepoint": 62181
     },
     {
         "displayName": "Hand",
         "iconName": "hand",
-        "tags": "gesture, fingers",
+        "tags": "gesture, fingers, raise hand",
         "group": "interface",
         "codepoint": 61920
     },
     {
         "displayName": "Chevron: backward",
         "iconName": "chevron-backward",
-        "tags": "skip, direction",
+        "tags": "skip, direction, k",
         "group": "interface",
         "codepoint": 61769
     },
     {
         "displayName": "Chevron: forward",
         "iconName": "chevron-forward",
-        "tags": "skip, direction",
+        "tags": "skip, direction, skip to end, strikeout",
         "group": "interface",
         "codepoint": 61771
     },
     {
         "displayName": "Rotate: document",
         "iconName": "rotate-document",
-        "tags": "turn, anti clockwise",
+        "tags": "turn, anti clockwise, revert file, reset file, return file, change side, flip, undo",
         "group": "editor",
         "codepoint": 62105
     },
     {
         "displayName": "Rotate: page",
         "iconName": "rotate-page",
-        "tags": "turn, anti clockwise",
+        "tags": "turn, anti clockwise, rotate, rotate counterclockwise, rotate left",
         "group": "editor",
         "codepoint": 62106
     },
     {
         "displayName": "Badge",
         "iconName": "badge",
-        "tags": "emblem, symbol, identification, insignia, marker",
+        "tags": "emblem, symbol, identification, insignia, marker, protect, privacy, shield, law enforcement, police, sheriff, rule enforcement, shield or protection, security",
         "group": "miscellaneous",
         "codepoint": 61743
     },
     {
         "displayName": "Grid view",
         "iconName": "grid-view",
-        "tags": "layout, arrangement",
+        "tags": "layout, arrangement, windows, boxes, squares, blocks, packing",
         "group": "editor",
         "codepoint": 61912
     },
     {
         "displayName": "Function",
         "iconName": "function",
-        "tags": "math, calculation",
+        "tags": "math, calculation, logic, fx",
         "group": "table",
         "codepoint": 61893
     },
     {
         "displayName": "Waterfall chart",
         "iconName": "waterfall-chart",
-        "tags": "graph, diagram",
+        "tags": "graph, diagram, waterfall, chart, bar chart, yield",
         "group": "data",
         "codepoint": 62230
     },
     {
         "displayName": "Stacked chart",
         "iconName": "stacked-chart",
-        "tags": "bar chart",
+        "tags": "bar chart, stacked bar graph, graph, sort, filter, chart, statistics, summary",
```

```suggestion
        "tags": "bar chart, graph, statistics, summary",
```

by this logic... we would add "statistics" to every chart icon... just something to think about re: consistency